xclim/indices/stats.py

```diff
@@ -121,7 +131,8 @@ def fit(
         output_dtypes=[float],
         keep_attrs=True,
         kwargs=dict(
-            dist=dc if method == "ML" else lm3dc,
+            # Don't know how APP should be included, this works for now
```

Comment:
A bit overkill for small data, but it works and might even be more performant than calling the `_fit_start` method directly for data with many chunks.

Reply:
Ok, sounds good.